<?php

namespace ContainerPxokFAr;

use Symfony\Component\DependencyInjection\Argument\RewindableGenerator;
use Symfony\Component\DependencyInjection\Exception\RuntimeException;

/**
 * @internal This class has been auto-generated by the Symfony Dependency Injection Component.
 */
class getSylius_Api_Validator_UniqueReviewerEmailService extends App_KernelDevDebugContainer
{
    /**
     * Gets the private 'sylius.api.validator.unique_reviewer_email' shared service.
     *
     * @return \Sylius\Bundle\ApiBundle\Validator\Constraints\UniqueReviewerEmailValidator
     */
    public static function do($container, $lazyLoad = true)
    {
        include_once \dirname(__DIR__, 4).'/vendor/symfony/validator/ConstraintValidatorInterface.php';
        include_once \dirname(__DIR__, 4).'/vendor/symfony/validator/ConstraintValidator.php';
        include_once \dirname(__DIR__, 4).'/vendor/sylius/sylius/src/Sylius/Bundle/ApiBundle/Validator/Constraints/UniqueReviewerEmailValidator.php';

        return $container->privates['sylius.api.validator.unique_reviewer_email'] = new \Sylius\Bundle\ApiBundle\Validator\Constraints\UniqueReviewerEmailValidator(($container->services['sylius.repository.shop_user'] ?? $container->load('getSylius_Repository_ShopUserService')), ($container->privates['sylius.api.context.user'] ?? $container->getSylius_Api_Context_UserService()));
    }
}
